modules/integration/tests/src/test/java/org/wso2/appmanager/integration/utils/VerificationUtil.java

```diff
@@ -0,0 +1,66 @@
+/*
+*Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
+*WSO2 Inc. licenses this file to you under the Apache License,
+*Version 2.0 (the "License"); you may not use this file except
+*in compliance with the License.
+*You may obtain a copy of the License at
+*
+*http://www.apache.org/licenses/LICENSE-2.0
+*
+*Unless required by applicable law or agreed to in writing,
+*software distributed under the License is distributed on an
+*"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+*KIND, either express or implied.  See the License for the
+*specific language governing permissions and limitations
+*under the License.
+*/
+
+package org.wso2.appmanager.integration.utils;
+
+
```

Comment:
Fixed in https://github.com/wso2/product-app-manager/commit/f021f86c3c398dddeeca17e225a37e42f7f7033b